### DISCUSSION TITLE
Downloading complete GO graph

### DISCUSSION THREAD

**Discussion by frenkiboy** [geneontology/helpdesk#353] [General]
Dear Madam or Sir, 

Could you please tell me where can I download the complete GO graph for Homo sapiens?
The files in .obo format seem to contain only GO term description and not the hierarchy.

Best regards, 

Vedran Franke

**Comment by suzialeksander** [comment]
Hi @frenkiboy,

I'm assuming you found the files at http://geneontology.org/docs/download-ontology.  If you were looking at go-basic.obo, it includes `is a`, `part of`, `regulates`, `negatively regulates` and `positively regulates`.  This is the version most compatible with outside tools.  We also have go.obo, which has `is a`, `part of`, `regulates`, `negatively regulates`, `positively regulates`, `has part`, `occurs_in`, and possibly others that can break older tools. 

If you want the most complete file, you would really want go-plus.owl, which can be opened in Protégé. 

Are you certain you're looking for an ontology file (which would contain all possible terms, not just the ones used for Human), instead of an annotation file (which would contain all terms annotated to Human, but not all terms in the GO)? If you're looking for all human annotations, you can find those on http://current.geneontology.org/products/pages/downloads.html
